src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,115 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import core.basesyntax.service.UserNotFoundException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 public class HelloWorldTest {
```

Comment:
It's a good practice to test critical cases for age: 17, 18, 19, for password "55555" "666666" "7777777". So check please if you have tasts for all this cases